docs/TuxOwneeClass.md

```diff
@@ -0,0 +1,250 @@
+Allows you to perform prop validation on the `nearestOwnerProps` to this `OwneeClass` component. For the full list of available properties that can be checked, please look at the [Reusable Components](http://facebook.github.io/react/docs/reusable-components.html) documenation.
+
+```javascript
+    owneeClassProps.nearestOwnerPropTypes = {
+      deleteMessage: React.PropTypes.function.isRequired,
+      editMessage: React.PropTypes.function.isRequired,
+    };
+```
+
+##### <a id="owneeClassProps-anyPropTypes"></a>Property - `owneeClassProps.anyPropTypes` - type: OBJECT - optional [#](#owneeClassProps-anyPropTypes)
+Allows you to perform prop validation on both the standard props of a component, as well as those props available via `nearestOwnerProps`. For the full list of available properties that can be checked, please look at the [Reusable Components](http://facebook.github.io/react/docs/reusable-components.html) documenation.
+
+```javascript
+    owneeClassProps.anyPropTypes = {
+      message: React.PropTypes.object.isRequired,
+      deleteMessage: React.PropTypes.function.isRequired,
+      editMessage: React.PropTypes.function.isRequired
+    };
+```
+
+**NOTE** When performing prop validations with `anyPropTypes` that props in `this.props` are validated first, so they will override the information within nearestOwnerProps.
+
+#### <a id="createOwneeClass-OwneeClass"></a>Return - `OwneeClass` - type: OBJECT [#](#createOwneeClass-OwneeClass)
+The `createOwneeClass` method returns a `TuxOwneeClass` which is an extension on top of a `React` Class. You can then simply use it in render methods as you would any other `React` Class.
+
+***
+
+##### <a id="TuxOwneeClass-Guidelines"></a>4) TuxOwneeClass Guidelines [#](#TuxOwneeClass-Guidelines)
+It is quite possible to never take advantage of the versatility that is provided by using the `nearestsOwnerProps`. Yet you would be missing out on the benefits that it provides. Passing properties down through a multitude of components is error prone, as it is easy to forget to pass all properties or methods that you need. Part of this difficulty can be alleviated by using `propTypes` to validate a components prop inputs.
+
+Using `nearestOwnerProps` allows you to know exactly what is available to an `OwneeClass`, without having to muddle through figuring out what will still be on that props object that you passed through React's `Spread` syntax. Therefore if you are passing methods from one child to the next only because you must expose those methods to their children, we recommend utilizing the `nearestOwnerProps` on the `OwneeClass`. It exposes the methods and properties of an `OwnerClass` via symantic and easy to reason about code.
+
+```javascript
+    //BAD
+    var React = require('tux/React');
+
+    // Passing properties down using the standard React prop syntax can be error prone, as you forget to pass a particular property into the right componet.
```

Comment:
Change to ...as you will have to manage and track sharing your methods throughout your component hierarchy.